var/spack/repos/builtin/packages/vtk/package.py

```diff
@@ -282,9 +282,8 @@ def cmake_args(self):
             # string. This fix was recommended on the VTK mailing list
             # in March 2014 (see
             # https://public.kitware.com/pipermail/vtkusers/2014-March/083368.html)
-            if (self.spec.satisfies('%clang') and
-                    self.compiler.is_apple and
-                    self.compiler.version >= Version('5.1.0')):
+            if (self.spec.satisfies('%apple-clang') and
```

Comment:
```suggestion
            if (self.spec.satisfies('%apple-clang@5.1.0:'):
```

Reply:
@adamjstewart I think if you want the suggestion to cover multiple lines you need to drag+click to highlight the included lines before creating the comment box.

Reply:
Yeah, I know, I was just being lazy

Reply:
Fixed